src/screens/EncounterRegistrationScreen.tsx

```diff
@@ -27,10 +26,9 @@ import {
   SectionHeader,
   Touchable,
   Wrapper,
-  WrapperRow,
-  WrapperWithOrientation
+  WrapperRow
 } from '../components';
-import { colors, consts, device, Icon, normalize, texts } from '../config';
+import { Icon, colors, consts, device, normalize, texts } from '../config';
```

Comment:
interesting, that `Icon` gets automatically organized first

Reply:
I tried it again, and it's still in that order.

Reply:
Sure :D How should it change after a few times? I was just wondering, manually i would have sorted it in as before.